pkg/payerreport/validation.go

```diff
@@ -0,0 +1,114 @@
+package payerreport
```

Comment:
I've included some of the validations in here, but there are going to be more once the merkle tree library lands